proc_IR.py

```diff
@@ -11,27 +11,35 @@
 clock_correction = 1.785
 filter_bandwidth = 5e3
 coh_sel = {'ph1':0,
-        'ph2':1}
+        'ph2':0}
```

Comment:
this doesn't make sense -- shouldn't `ph2` be the 90 pulse and actually have Δp=-1?

Reply:
should be fixed